std/assembly/date.ts

```diff
@@ -1,6 +1,20 @@
 import { E_VALUEOUTOFRANGE } from "util/error";
 import { now as Date_now } from "./bindings/Date";
 
+// @ts-ignore: decorator
+@inline const
+  MILLIS_PER_DAY    = 1000 * 60 * 60 * 24,
+  MILLIS_PER_HOUR   = 1000 * 60 * 60,
+  MILLIS_PER_MINUTE = 1000 * 60,
+  MILLIS_PER_SECOND = 1000;
+
+// ymdFromEpochDays returns values via globals to avoid allocations
+// @ts-ignore: decorator
+@lazy let
+  year: i32,
```

Comment:
Can perhaps reduce by one global here, by returning say `year` from the function